src/content/App.js

```diff
@@ -40,7 +40,7 @@ class App extends Component {
 
         <div className="right-panel">
           {
-            this.state.book && <Book key={this.state.book.key} data={this.state.book} />
```

Comment:
I removed `this.state.book` because it doesn't seem like it was doing anything? I think?

Reply:
You need to define the state of the book or this happens 
![2019-08-20_09-20-58](https://user-images.githubusercontent.com/8514591/63355503-dbca7780-c32b-11e9-9707-a22570bc1190.png)
^_^